app/views/tasks/create.js.coffee

```diff
@@ -18,15 +18,27 @@
 # INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-<% if @errors.present? %>
-$("#modal_errors").html("<%= escape_javascript(render(partial: 'modal_errors', locals: {errors: @errors})) %>")
+<% if @errors %>
+$("[name^='task']:not([type='hidden'])").parents('.form-group').removeClass('is-invalid').addClass('is-valid')
+$('.form-error').remove()
+<% @errors.messages.each do |attr, messages| %>
+<% messages.each do |message| %>
+$("[name='task[<%= attr.to_s %>]']").parents('.form-group').removeClass('is-valid').addClass('is-invalid').append("<small class='form-text form-error'><%= message.capitalize.html_safe %></small>")
+<% end %>
+<% end %>
 <% else %>
-if !$('.notification.task-notifications').length
-  $('<span class="notification task-notifications"></span>').appendTo($('a.tasks'))
-$(".notification.task-notifications").empty().append("<%= current_identity.reload.tasks_count %>")
-$("#flashContainer").html("<%= escape_javascript(render('flash')) %>")
-$('#task-list').bootstrapTable('refresh', {silent: "true"})
-$("#modalContainer").modal 'hide'
+# TODO: check that notifications update
+$(".notification-badge.task-notifications").empty().append("<%= current_identity.reload.tasks_count %>")
```

Comment:
Instead of using all of this javascript for the notification bubble, would it be easier to put the navbar tasks link in a partial then re-render the partial?